tests/TASVideos.MovieParsers.Tests/TASVideos.MovieParsers.Tests.csproj

```diff
@@ -18,209 +19,30 @@
-		<EmbeddedResource Include="LsmvSampleFiles\norerecordentry.lsmv" />
-		<EmbeddedResource Include="LsmvSampleFiles\savestate.anchor.lsmv" />
-		<EmbeddedResource Include="LsmvSampleFiles\savestate.lsmv" />
-		<EmbeddedResource Include="LtmSampleFiles\2frames.ltm" />
-		<EmbeddedResource Include="LtmSampleFiles\annotations.ltm" />
-		<EmbeddedResource Include="LtmSampleFiles\arcade.ltm" />
-		<EmbeddedResource Include="LtmSampleFiles\dos.ltm" />
-		<EmbeddedResource Include="LtmSampleFiles\flash-extrachars-linuxfallback.ltm" />
-		<EmbeddedResource Include="LtmSampleFiles\flash.ltm" />
-		<EmbeddedResource Include="LtmSampleFiles\linux.ltm" />
-		<EmbeddedResource Include="LtmSampleFiles\macos.ltm" />
-		<EmbeddedResource Include="LtmSampleFiles\noframerate.ltm" />
-		<EmbeddedResource Include="LtmSampleFiles\pc98.ltm" />
-		<EmbeddedResource Include="LtmSampleFiles\pico8.ltm" />
-		<EmbeddedResource Include="LtmSampleFiles\savestate.ltm" />
-		<EmbeddedResource Include="LtmSampleFiles\unknown-linuxfallback.ltm" />
-		<EmbeddedResource Include="LtmSampleFiles\variableframerate.ltm" />
-		<EmbeddedResource Include="LtmSampleFiles\windows.ltm" />
-		<EmbeddedResource Include="M64SampleFiles\2frames.m64" />
-		<EmbeddedResource Include="M64SampleFiles\pal.m64" />
-		<EmbeddedResource Include="M64SampleFiles\savestate.m64" />
-		<EmbeddedResource Include="M64SampleFiles\sram.m64" />
-		<EmbeddedResource Include="M64SampleFiles\wrongheader.m64" />
-		<EmbeddedResource Include="MarSampleFiles\2frames.mar" />
-		<EmbeddedResource Include="MarSampleFiles\noframerate.mar" />
-		<EmbeddedResource Include="MarSampleFiles\wrongheader.mar" />
-		<EmbeddedResource Include="OmrSampleFiles\2seconds.omr" />
-		<EmbeddedResource Include="OmrSampleFiles\msxnewerformat.omr" />
-		<EmbeddedResource Include="OmrSampleFiles\coleco.omr" />
-		<EmbeddedResource Include="OmrSampleFiles\coleconewerformat.omr" />
-		<EmbeddedResource Include="OmrSampleFiles\pal.omr" />
-		<EmbeddedResource Include="OmrSampleFiles\savestate.omr" />
-		<EmbeddedResource Include="OmrSampleFiles\svi.omr" />
-		<EmbeddedResource Include="OmrSampleFiles\svinewerformat.omr" />
-		<EmbeddedResource Include="OmrSampleFiles\sg1000.omr" />
-		<EmbeddedResource Include="TasprojSampleFiles\branch.tasproj" />
-		<EmbeddedResource Include="VbmSampleFiles\2frames.vbm" />
-		<EmbeddedResource Include="VbmSampleFiles\gb.vbm" />
-		<EmbeddedResource Include="VbmSampleFiles\gbc.vbm" />
-		<EmbeddedResource Include="VbmSampleFiles\savestate.vbm" />
-		<EmbeddedResource Include="VbmSampleFiles\sgb.vbm" />
-		<EmbeddedResource Include="VbmSampleFiles\sram.vbm" />
-		<EmbeddedResource Include="VbmSampleFiles\wrongheader.vbm" />
-		<EmbeddedResource Include="WtfSampleFiles\2frames.wtf" />
-		<EmbeddedResource Include="WtfSampleFiles\noframerate.wtf" />
-		<EmbeddedResource Include="WtfSampleFiles\wrongheader.wtf" />
+		<EmbeddedResource Include="Bk2SampleFiles\*.bk2" />
```

Comment:
Wow, didn't know you could do that, that makes life so much easier since everyone forgets to add their files to the project in their PRs *thumbs up*